server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4554,6 +4571,71 @@ public void finalizeStop(VirtualMachineProfile profile, Answer answer) {
         return vmParamPair;
     }
 
+    private void validateDeploymentDestination(Pod destinationPod, Cluster destinationCluster, Host destinationHost) {
```

Comment:
I would rather have that the user would be able to pass either a pod, or a cluster, or a host. the smaller implies the bigger and makes it redundant as parameter.

Reply:
This validation is not necessary since we take the lowest unit supplied and use that for the deployment. 

Check has been removed